### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

**Review by Yaroslav-Stetsko — APPROVED** [review]
Great gob!

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,48 @@
...
+            if user_input == "stop":
+                file.write("\n")
+                break
+
+            line_counter += 1
+            file.write(f"{line_counter} {user_input}\n")
+
+
+if __name__ == "__main__":
+    get_from_terminal()
```
Comment (Y-Havryliv) [inline app/create_file.py:48]: try to test youd code when creating multiple directories with `-d` flag

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: app/create_file.py (hunk 1 above)
Comment (HStakhiv) [reply app/create_file.py:48]: Like this `python app/create_file.py -d dir1 dir2 dir3 d4 d5 d6 `?

Comment (HStakhiv) [reply app/create_file.py:48]: That's right, I lost one folder ((

Comment (Y-Havryliv) [reply app/create_file.py:48]: all good, nice job

Comment (HStakhiv) [reply app/create_file.py:48]: Thanks

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**